.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "common/OSCAL"]
```

Comment:
@wendellpiez, thanks for framing this. As we discussed in Gitter, I do not have a strong position on approach or the other, but I see this dichotomy. I would frame this as an important question that needs answering:

> Does oscal-xslt presume a specific version of Metaschema and metaschema-xslt as it is bound to a particular version of OSCAL?

I use the word "version" loosely here, but because of the submodule commit pinning, it will be a particular assumed version. It is convenient, by dint of code organization, to side-step that question by directly adding metaschema-xslt and not transitively through OSCAL, but this is the decision worth deciding. I would presume there is some need to code oscal-xslt, this repo, against the underlying assumptions of OSCAL, not just metaschema-xslt, but definitely that. And if you need to change that version of OSCAL and assumptions, you should be moving around an OSCAL commit (including the submodule of metaschema-xslt within in your fork if you are developing a change and want to track it).